docs/user-guide/work-packages/baseline-comparison/README.md

```diff
@@ -0,0 +1,121 @@
+
+## Understanding the comparison results
+
+After selecting the comparison point, OpenProject presents the comparison results in table view using three icons to indicate the type of change for each work package.
+
+![Baseline enabled showing changes to the work package table](13-0_Baseline_table.png)
+
+When Baseline is enabled, you will see a legend at the top of the page which shows:
+
+![A legend is visible on top of the table when Baseline is enabled](13-0_Baseline_legend.png)
+
+- The comparison point or comparison period
+- The number of work packages added to view
+- The number of work packages removed from view
+- The number of work packages were modified
+
+
+### Change icons
+
+#### Now meets filter criteria
+
+![Icon](13-0_Baseline_nowMeets.png)
+
+Work packages that meet the filter criteria now but did not in the past are marked with an "Added" icon. These work packages were added to the current query after the selected comparison point, either because they were newly created since then or certain attributes changed such that they meet the filter criteria. 
+
+> **Note**: These do not necessarily represent _newly created_ work packages; simply those that are new to this particular view.
+
+#### No longer meets filter criteria
+
+![Icon](13-0_Baseline_noLongerMeets.png)
+
+Work packages that no longer meet the filter criteria now are marked with a "Removed" icon. These work packages were filtered out within the comparison period.
+
+>**Note**: These do _not_ represent deleted work packages, which are not visible at all since no history of deleted work packages is maintained. Deleted work packages are simply ignored by Baseline.
+
+#### Maintained with changes
+
+![Icon](13-0_Baseline_maintainedChanges.png)
+
+Work packages that meet the filter criteria now (and also did at the comparison point) but which have undergone changes in certain attributes are marked with a "Modified" icon.
+
+#### No changes
+
+When there are no changes to a work package in the comparison period, no icon is shown.
+
+### Old values
+
+When changes in the comparison period concern attributes that are visible as columns in the work package table, Baseline will show both old and current values. If the attribute you are interested in is not visible, you will need to [add it as a column](../work-package-table-configuration). 
+
+![Old values are visible in the work package table view](13-0_Baseline_oldNewValues.png)
+
+Each attribute that has changed will have a grey background, with the old value crossed out visible on the top left corner of the cell, above the new value.
+
+This allows you to have a complete view of what has changed in the comparison period.
+
+>**Note**: Some attributes that are not tracked by Baseline are not able to show the old values in the work package table. If any of the columns in your work package table are not tracked, a small warning in the column header will warn you of this.
```

Comment:
Do we want to create a list of the non tracked attributes or is not needed?

Reply:
Maybe mention an example of such attribute though?

Reply:
You're right, that might help; added examples in the latest commit.